### ISSUE TITLE
PDF JIS Z 8301: commentary section page numbering

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/metanorma-jis#30]
Source issue: https://github.com/metanorma/metanorma-jis/issues/9

The last section has additional own page numbering starts from 1.
Original PDF (end of first page and start of 2nd):
![image](https://user-images.githubusercontent.com/5559926/228205229-baa5a1f0-6ccf-4ba0-a26a-a250cd448324.png)

**Comment by ronaldtse** [comment]
This depends on the implementation of:
* [ ] #28

**Comment by Intelligent2013** [comment]
The second page numbering in the source https://github.com/metanorma/mn-samples-jis/blob/main/reference-docs/TR-X-0067-2002/tr_x_0067(dvd-r).Doc made with using the calculated field:
![image](https://user-images.githubusercontent.com/5559926/231541314-3943317d-2b7f-4d6f-8790-830c551da5ff.png)

- `PAGE` means the current page number
-  `DOCPROPERTY "解説P"` from the document properties:
![image](https://user-images.githubusercontent.com/5559926/231542446-e20e8a91-f068-4782-adc6-d18ab9425805.png)

I don't figure out how to realize the multiple page numbering in the XSL-FO. Only one more IF XML complicated post-processing step.

**Comment by ronaldtse** [comment]
This is a detail that @opoudjis would want to know.

**Comment by opoudjis** [comment]
Yeah, I found that out. It is dirty and cheating, but fortunately I generate the document, so I did a workaround which Word allowed me: you can retrieve the page number of an arbitrary anchor, so I inserted a bookmark on the preceding page, and set page numbering for this instance to add the page number of that bookmark.

I don't know if you can do that kind of dynamic arithmetic, but the point is, while Word does in theory support two page numberings (of current section and current document), that was a recent addition to Word, and in practice, you need to do arithmetic like this on the page number display.

https://github.com/metanorma/metanorma-jis/issues/63

**Comment by Intelligent2013** [comment]
To re-calculate commentary page number just enough to count the preceding pages with `<id name="_independent_page_number_commentary"/>` inside, for instance:
```xml
		<viewport transform="translate(62362,772441)" width="470551" height="69448" region-type="Footer">
			<viewport width="470551" height="68031" region-type="Footer">
				<font family="IPAexGothic" size="9000"/>
				<id name=""/>
				<text x="224274" y="8815" foi:struct-ref="5f4">解</text>
				<font family="Times New Roman" weight="700"/>
				<id name="_independent_page_number_commentary"/>
				<text x="241777" y="8815" foi:struct-ref="5f5">9</text>
				<font family="IPAexMincho" weight="400" size="7000"/>
				<id name=""/>
				<text x="137275" y="30511" foi:struct-ref="5f6">著作権法により無断での複製，転載等は禁⽌されております。</text>
			</viewport>
		</viewport>
	</content>
	<page-trailer>
		<nav:goto-xy id="___x89e3___x8aac_" page-index="13" x="62362" y="140596"/>
	</page-trailer>
</page>
```
To do: replace value in `<text x="241777" y="8815" foi:struct-ref="5f5">9</text>`.

**Comment by Intelligent2013** [comment]
Done:
![image](https://github.com/metanorma/metanorma-jis/assets/5559926/4600641b-8089-4f71-8571-f51ae1f5c0da)

Updated:
- JIS XSLT
- `mn2pdf` (v1.73)
